components/Navbar.tsx

```diff
@@ -4,17 +4,36 @@ import {
   Divider,
   Drawer,
   IconButton,
+  SvgIcon,
   Link as MUILink,
   Menu,
   MenuItem,
   Toolbar,
 } from "@mui/material";
-import { Menu as MenuIcon, MoreVert } from "@mui/icons-material";
+import {
+  Menu as MenuIcon,
+  MoreVert,
+  GetApp,
+  AutoAwesome as AutoAwesomeIcon,
+  VolunteerActivism as VolunteerActivismIcon,
+  Science as ScienceIcon,
+  Article as ArticleIcon,
+  GitHub as GitHubIcon,
+  Groups as GroupIcon,
+  GroupAdd as GroupAddIcon,
+  Reddit as RedditIcon,
+  Facebook as FacebookIcon,
+  Telegram as TelegramIcon,
+  Twitter as TwitterIcon,
+  Instagram as InstagramIcon,
+  Code as CodeIcon,
+} from "@mui/icons-material";
 import { useCallback, useState } from "react";
 import styled, { css, keyframes, useTheme } from "styled-components";
 import useScrollPosition from "@react-hook/window-scroll";
 import { SkipNavContent, SkipNavLink } from "@reach/skip-nav";
 import "@reach/skip-nav/styles.css";
+import DiscordLogo from "/public/images/logos/discord-logo.svg";
```

Comment:
instead of this, we should create a new file under `/components` named `Icons.tsx` and we paste all external SVGS in there

heres an example

Icons.tsx
```tsx
export const DiscordLogo = (props: SVGProps<SVGSVGElement>) => (
  <svg {...props}></svg>
)

````